### PR TITLE
fix: articles load more button cache issues

### PR DESCRIPTION
**Pull request by melniiv**
### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

**Comment by terovirtanen** [comment]
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr360.dev.hel.ninja** :rocket::rocket::rocket:

File: apps/events-helsinki/src/pages/articles/index.tsx [hunk 1]
```diff
@@ -65,6 +65,10 @@ export default function ArticleArchive({
     networkStatus,
   } = usePostsQuery({
     notifyOnNetworkStatusChange: true,
+    // does not work without fetchPolicy: 'network-only',
+    fetchPolicy: 'network-only',
+    // not sure if nextFetchPolicy is needed
+    nextFetchPolicy: 'cache-first',
```
Comment (nikomakela) [inline apps/events-helsinki/src/pages/articles/index.tsx:71]: This looks OK to me.

fetchPolicy is used every time the component mounts. NextFetchPolicy is used for every query execution after the component mounts. 

https://www.apollographql.com/docs/react/data/queries/#nextfetchpolicy
> You can also specify a query's nextFetchPolicy. If you do, fetchPolicy is used for the query's first execution, and nextFetchPolicy is used to determine how the query responds to future cache updates

**Review by nikomakela — APPROVED** [review]

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr360.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr360.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr360.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr360.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr360.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**EVENTS GRAPHQL PROXY is deployed to platta: https://events-graphql-proxy-pr360.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**